todo_app/tests_e2e/test_ui.py

```diff
@@ -46,6 +41,49 @@ def driver():
     with webdriver.Chrome(options=options) as driver:
         yield driver
 
+def test_user_admin(driver, test_app):
+    before_XPath = "//*[@id='users']/tbody/tr["
+    aftertd_XPath = "]/td["
+    aftertr_XPath = "]"
+
+
+    driver.get('http://localhost:5000/users')
+
+    assert driver.title == 'To-Do App'
+
+    num_rows = len (driver.find_elements_by_xpath("//*[@id='users']/tbody/tr"))
+
+    assert num_rows == 4, "Should be 4 rows in the table"
```

Comment:
Is this based on some pre-configured data for the test database? This test fails locally for me, so I'm guessing so. It would be nice to have the test insert that data if it doesn't already exist